### PR TITLE
Fixed #28050 - Added template path to TemplateSyntaxError exception.

### PR DESCRIPTION
**Pull request by rogeremasse**
Adjust template exception test to account for template path. 

Here's an example technical_500.html render with the duplication removed:

template-syntax-error
Remove redundant template path and line number in technical_500.html.

### CODE REVIEW

File: django/template/base.py [hunk 1]
```diff
@@ -197,6 +197,9 @@ def compile_nodelist(self):
         try:
             return parser.parse()
         except Exception as e:
+            if self.origin.name != UNKNOWN_SOURCE:
+                template_path = "Template : %s, " % self.origin.name
```
Comment (Kangaroux) [inline django/template/base.py]: Small nitpick, can you remove the space? `Template:`

Comment (rogeremasse) [reply django/template/base.py]: Sure thing.  Catching up and fixing tests. Update to PR forthcoming.

**Comment by rogeremasse** [comment]
I'm maybe a bit confused about the proper gitflow. Apologies.

I merged rogeremasse/django with the latest master branch from django/django using:

  774  git remote add upstream https://github.com/django/django
  775  git remote -v
  776  git fetch upstream
  777  git checkout master
  778  git merge upstream/master

Then I merged my previous patch into my current ticket branch:

780  git checkout ticket_28050
785  git merge origin/ticket_28050

Then I also merged changes to the master into my ticket branch:

  781  git merge master

However, that results in my pull request showing all the changes (not just mine) since the PR was created. Is that normal?

**Comment by Kangaroux** [comment]
@rogeremasse It should just say "Merged master into <branch>". 

To clean things up you can do this:

```bash
# Update master
git checkout master
git pull origin master

# Create a new branch for your ticket
git checkout -b ticket-28050-2

# Cherry pick the commits you made
git cherry-pick <commit>
git cherry-pick <commit>
...

# Push to github
git push -u origin ticket-28050-2
```

**Pull request closed without merging**